scripts/clear-docker

```diff
@@ -1,6 +1,11 @@
 #!/usr/bin/env bash
 set -euxo pipefail
 
+HERE=$(realpath "$(dirname $0)")
+. "$HERE"/common
+
 docker rm --force $(docker ps --all --quiet) || true
 docker network prune --force
 docker volume prune --force
+
+docker volume rm $ORDERLY_VOLUME $ORDERLY_REMOTE_VOLUME
```

Comment:
probably best not to remove every docker container and network and volume.... if people are working on other stuff its gets removed too ahha.. i learnt this the hard way by running this hahaha

Reply:
yep fair enough, can modify it so it removes just the containers we spin up